src/components/Settings/Settings.test.js

```diff
@@ -45,13 +45,13 @@ describe('Settings', () => {
   it('confirms when account settings were changed successfully', async () => {
     const mockedRequest = mockUpdateUserSuccess(fakeSettingsData);
 
-    const { getByTestId, queryByText } = render(<Settings />, {
+    const { getByLabelText, getByTestId, queryByText } = render(<Settings />, {
       state: fakeState,
     });
 
-    const firstName = getByTestId('firstName-input');
-    const lastName = getByTestId('lastName-input');
-    const locale = getByTestId('locale-input');
+    const firstName = getByLabelText('First Name');
```

Comment:
what about using getByRole? https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-the-wrong-query

Reply:
Nice!!!

Reply:
I think that for inputs we should keep the getByLabelText query since it actually validates that the label of each input is properly associated to it. What do you think?

Reply:
You are right 👌